aws.go

```diff
@@ -114,12 +143,14 @@ func (a *AWS) Upload(local, remote string) error {
 			Key:         aws.String(remote),
 			Body:        file,
 			ContentType: aws.String(contentType),
+			ContentEncoding: aws.String(contentEncoding),
```

Comment:
I'm afraid I don't understand. Will this be backwards compatible if contentEncoding is empty?